### PR TITLE
[Docs] Linux Service Edit

### PR DESCRIPTION
**Pull request by bagasme**
Per title:

- Rename the `h3` title to be more generic
- Invite to edit config files, instead of running `sudo vim`
- Use `systemctl enable --now` instead of running `systemctl enable` and `systemctl start` separately

### CODE REVIEW

File: docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md [hunk 1]
```diff
@@ -31,8 +28,7 @@ PORT or remove the -p flag if default port is used.
 
 Enable and start Gitea at boot:
 ```
-sudo systemctl enable gitea
-sudo systemctl start gitea
+sudo systemctl enable gitea --now
```
Comment (6543) [inline docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md]: @bagasme  eaven if its a shortcut; I would leave it so people understand what they do

File: docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md [hunk 2]
```diff
@@ -49,19 +45,14 @@ Create a log dir for the supervisor logs:
...
-environment. Change the PORT or remove the -p flag if default port is used.
+Using your favorite editor (such as `vim`), change the user (git) and home
+(/home/git) settings to match the deployment environment. Change the PORT
+or remove the -p flag if default port is used.
 
 Lastly enable and start supervisor at boot:
 ```
-sudo systemctl enable supervisor
-sudo systemctl start supervisor
+sudo systemctl enable supervisor --now
```
Comment (6543) [inline docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md:68]: same as above

**Review by 6543 — CHANGES_REQUESTED** [review]
shortcuts are good for those who already know ...

File: docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md (hunk 2 above)
Comment (bagasme) [reply docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md:68]: @6543 basically `systemctl enable --now` enables the service and immediately starts it

File: docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md [hunk 3]
```diff
@@ -31,8 +28,10 @@ PORT or remove the -p flag if default port is used.
 
 Enable and start Gitea at boot:
 ```
-sudo systemctl enable gitea
-sudo systemctl start gitea
+sudo systemctl enable gitea && sudo systemctl start gitea
```
Comment (6543) [inline docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md]: I realy would just leave it as it is - I know you you like to give gitea users a hint about the `--now` function but  
but I think we should keep it simple (dont make changes to the `Enable and start Gitea at boot:` section)

beside this I think this are good chagnes

Comment (6543) [reply docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md]: -> do you have a reason to expose the `sudo systemctl enable gitea --now` in our docs?

Comment (bagasme) [reply docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md]: @6543 just shorcut to enable and start the services

Comment (6543) [reply docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md]: I personaly would leave it as it is now ...

**Review by 6543 — APPROVED** [review]

**Review by sapk — COMMENTED** [review]
~I don't think that the ``~ miss click will redo the review. sorry.

File: docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md [hunk 4]
```diff
@@ -13,16 +13,13 @@ menu:
...
+You can run Gitea as service, using either systemd or supervisor. The steps below tested on Ubuntu 16.04, but those should work on any Linux distributions (with little modification).
 
-Run the below command in a terminal:
-```
-sudo vim /etc/systemd/system/gitea.service
-```
+#### Using systemd
 
-Copy the sample [gitea.service](https://github.com/go-gitea/gitea/blob/master/contrib/systemd/gitea.service).
+Copy the sample [gitea.service](https://github.com/go-gitea/gitea/blob/master/contrib/systemd/gitea.service) to `/etc/systemd/system/gitea.service`, then edit the file with your favorite editor (such as `vim`).
```
Comment (sapk) [inline docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md]:
```suggestion
Copy the sample [gitea.service](https://github.com/go-gitea/gitea/blob/master/contrib/systemd/gitea.service) to `/etc/systemd/system/gitea.service`, then edit the file with your favorite editor.
```

File: docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md [hunk 5]
```diff
@@ -49,19 +50,20 @@ Create a log dir for the supervisor logs:
...
-sudo vim /etc/supervisor/supervisord.conf
-```
-
 Append the configuration from the sample
-[supervisord config](https://github.com/go-gitea/gitea/blob/master/contrib/supervisor/gitea).
+[supervisord config](https://github.com/go-gitea/gitea/blob/master/contrib/supervisor/gitea) to `/etc/supervisor/supervisord.conf`.
 
-Change the user (git) and home (/home/git) settings to match the deployment
-environment. Change the PORT or remove the -p flag if default port is used.
+Using your favorite editor (such as `vim`), change the user (git) and home
```
Comment (sapk) [inline docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md]:
```suggestion
Using your favorite editor, change the user (git) and home
```

**Review by sapk — COMMENTED** [review]
The `such as vim` doesn't add any value to the docs. Users that edit service file should know what is an editor.

**Review by sapk — APPROVED** [review]

**Pull request merged**